Vagrantfile

```diff
@@ -104,49 +104,66 @@ Vagrant.configure(2) do |config|
     mount -t overlayfs overlayfs -o lowerdir=#{overlay_lower},upperdir=#{overlay_upper} #{overlay_mount}
   SCRIPT
 
-  # run install script
-  plugin_string = plugins.join(" ")
-  config.vm.provision "shell", inline: <<-SCRIPT
-    if [ ! -f /var/local/reddit_installed ]; then
-      echo "running install script"
-      cd /home/#{vagrant_user}/src/reddit
-      REDDIT_PLUGINS="#{plugin_string}" REDDIT_DOMAIN="#{hostname}" ./install-reddit.sh
-      touch /var/local/reddit_installed
-    else
-      echo "install script already run"
-    fi
-  SCRIPT
-
-  # setup private code
-  if File.exist?("#{code_share_host_path}/private/vagrant_setup.sh")
-    config.vm.provision "shell",
-      path: "#{code_share_host_path}/private/vagrant_setup.sh",
-      args: [vagrant_user]
+  config.vm.define "travis" do |travis|
```

Comment:
Can you add a comment here or at the top of the file explaining that this is never run, but is to try to replicate what will happen in travis?